lib/style.css

```diff
@@ -179,20 +179,72 @@ td.feed-source-info {
 
 /* feed validation error types */
 
-.gtfs-error-high {
+.gtfs-error-HIGH-label {
```

Comment:
Are these errors to circumvent the limited colors available out of the box in the version of react-bootstrap we are using?

Reply:
Yea, I believe that's the intent. There may be some disagreement between the react-bs colors and these though, so we might want to reconsider the mixing of the bs label colors with these new ones. I do like the pattern of using the priority id in the class name selector rather than using a conditional to select the color within the code.